traitcuration/traits/managers.py

```diff
@@ -0,0 +1,35 @@
+from django.contrib.auth.base_user import BaseUserManager
+from django.utils.translation import ugettext_lazy as _
+
+
+class CustomUserManager(BaseUserManager):
+    """
+    Custom user model manager where email is the unique identifiers
+    for authentication instead of usernames.
```

Comment:
This can fit into one line <120 characters. Looks like pylint strikes again :)

```suggestion
    Custom user model manager where email is the unique identifier for authentication instead of usernames.
```